include/deal.II/grid/manifold_lib.h

```diff
@@ -127,6 +116,131 @@ class SphericalManifold : public ChartManifold<dim, spacedim, spacedim>
 
 
 /**
+ * Manifold description for a spherical space coordinate system.
+ *
+ * You can use this Manifold object to describe any sphere, circle,
+ * hypersphere or hyperdisc in two or three dimensions, both as a co-dimension
+ * one manifold descriptor or as co-dimension zero manifold descriptor.
+ *
+ * The two template arguments match the meaning of the two template arguments
+ * in Triangulation<dim, spacedim>, however this Manifold can be used to
+ * describe both thin and thick objects, and the behavior is identical when
+ * dim <= spacedim, i.e., the functionality of SphericalManifold<2,3> is
+ * identical to SphericalManifold<3,3>.
+ *
+ * While PolarManifold reflects the usual notion of polar coordinates,
+ * it may not be suitable for domains that contain either the north
+ * pole, the south pole or the center.  Consider for instance the pair
+ * of points \f$x_1=(1,\pi/3,0)\f$ and \f$x_2=(1,\pi/3,\pi)\f$ in
+ * polar coordinates (laying on the surface of a sphere with radius
+ * one, on a parallel at height $\pi/3$. In this case connecting the
+ * points with a straight line in polar coordinates would take the
+ * long road around the globe, without passing through the north pole.
+ *
+ * These two points would be connented (using a PolarManifold) by the curve
+ * \$[
+ * \begin{align}
+ *   s: [0,1]  & \rightarrow &  \mathbb S^3 \\
+ *           t & \mapsto     &  (1,\pi/3,0) + (0,0,t\pi)
+ * \$]
+ * This curve is not a geodesic on the sphere, and it is not how we
+ * would connect those two points. A better curve, would be the one
+ * passing through the North pole:
+ * \[
+ *  s(t) = x_1 \cos(\alpha(t)) + \kappa \times x_1 \sin(\alpha(t)) +
+ *  \kappa ( \kappa \cdot x_1) (1-\cos(\alpha(t))).
+ * \]
+ * where $\kappa = \frac{x_1 \times \x_2}{\Vert x_1 \times \x_2 \Vert}$
+ * and $\alpha(t) = t * \arccos(x_1 * x_2) $ for $t\in[0,1]$.
+ * Indeed, this is a geodesic, and it is the natural choice when
+ * connecting points on the surface of the sphere.
```

Comment:
I *think* what you're trying to say with this example is that the latter is what the current class actually implements. Maybe say somewhere in this example that the former is what `PolarManifold` implements, and the latter what `SphericalManifold` implements?